dataprofiler/tests/profilers/test_int_column_profile.py

```diff
@@ -444,6 +513,8 @@ def test_profile_merge_edge_case(self):
         profiler = profiler3 + profiler4
         self.assertEqual(profiler.min, 2)
         self.assertEqual(profiler.max, 5)
+        self.assertEqual(profiler.skewness, 0)
```

Comment:
can we use np or pd to calculate the value `0`, and same as below

Reply:
@AnhTruong using pd / np would not make sense in this case as skew / kurtosis would not be able to be calculated on such small basis. We would have to look into an online algorithm for single values and to adapt it as such.

```
def online_kurtosis(data):
    n = mean = M2 = M3 = M4 = 0

    for x in data:
        n1 = n
        n = n + 1
        delta = x - mean
        delta_n = delta / n
        delta_n2 = delta_n * delta_n
        term1 = delta * delta_n * n1
        mean = mean + delta_n
        M4 = M4 + term1 * delta_n2 * (n*n - 3*n + 3) + 6 * delta_n2 * M2 - 4 * delta_n * M3
        M3 = M3 + term1 * delta_n * (n - 2) - 3 * delta_n * M2
        M2 = M2 + term1

    # Note, you may also calculate variance using M2, and skewness using M3
    # Caution: If all the inputs are the same, M2 will be 0, resulting in a division by 0.
    kurtosis = (n * M4) / (M2 * M2) - 3
    return kurtosis
```
from https://en.wikipedia.org/wiki/Algorithms_for_calculating_variance#Higher-order_statistics